py/torch_tensorrt/dynamo/converter_registry.py

```diff
@@ -0,0 +1,266 @@
+from dataclasses import dataclass, field
+from typing import Any, Callable, Dict, Optional, Sequence, Union
+
+from torch.fx.node import Target, Node
+from torch_tensorrt.fx.converter_registry import CONVERTERS
+
+
+@dataclass(frozen=True)
+class GenericNode:
+    """Class representing a generic Torch FX Node
+
+    Convenience class for when we do not have access to the node,
+    and instead only its target, args, and kwargs, as is the case
+    in the TRTInterpreter
+    """
+
+    target: Any
+    args: Sequence[Any]
+    kwargs: Dict[str, Any]
+
+
+# Defines the different types of valid nodes
+NodeKinds = Union[GenericNode, Node]
+
+
+@dataclass(frozen=True)
+class ConverterSupport:
+    """Class representing a converter implementation and support function
+
+    Args:
+        converter_implementation: Function which converts said node to a TRT equivalent
+        check_args: Function which takes in a NodeKind and returns a bool indicating
+            whether that node can be supported by its companion converter. Note that
+            this function must only access the .target, .args, and .kwargs fields
+            of the node, and cannot modify the node or its graph
+    """
+
+    converter_implementation: Callable
+    check_args: Callable[[NodeKinds], bool] = field(default=lambda node: True)
+
+
+# Dictionary representing Dynamo aten-only converters
+# Each converter maps to a sequence of at least one ConverterSupport object(s)
+DYNAMO_ATEN_CONVERTERS: Dict[Target, Sequence[ConverterSupport]] = {}
+
+
+def dynamo_tensorrt_converter(
+    key: Target,
+    enabled: bool = True,
+    check_args: Optional[Callable[[NodeKinds], bool]] = None,
```

Comment:
Capability validator - rename the above (`capability_validator`)

There may be cases which your converter cannot handle, which we can validate at partitioning time. For instance, cannot support a specific setting.

Reply:
`high_priority: bool` flag/argument addition, insert converter at the start of the sequence of converters to check + use first.